examples/multiple_viewers_.py

```diff
@@ -13,12 +13,14 @@
 
 # add the image
 photographer = data.camera()
-viewer_a = napari.view_image(photographer, name='photographer')
+viewer_a = napari.Viewer()
```

Comment:
```suggestion
viewer_a = napari.Viewer(title="viewer a")
```
Totally optional, just noticed we don't do this.